PocketKit/Sources/PocketKit/MyList/ItemsList/SwiftUI/ListItem.swift

```diff
@@ -27,7 +29,17 @@ struct ListItem: View {
                 ItemTags(tags: viewModel.item.tags, tagCount: viewModel.item.tagCount)
                 Spacer()
                 ActionButton(viewModel.favoriteAction(index: viewModel.index, scope: scope), selected: viewModel.isFavorite)
-                ActionButton(viewModel.shareAction)
+
+                ActionButton(viewModel.shareAction(index: viewModel.index, scope: scope))
+                    .sheet(isPresented: $viewModel.presentShareSheet) {
+                        if #available(iOS 16.0, *) {
+                            ShareSheetView(activity: PocketItemActivity(url: viewModel.item.url))
+                                .presentationDetents([.medium])
+                        } else {
+                            ShareSheetView(activity: PocketItemActivity(url: viewModel.item.url))
```

Comment:
issue where the share sheet takes up the whole screen for iOS 15

Reply:
adding comment as weird bug, convo here: https://pocket.slack.com/archives/C01UH57EJKD/p1677166224894789